src/Playwright.Tests/BrowserTypeConnectTests.cs

```diff
@@ -80,21 +80,70 @@ public void TearDown()
         }
 
         [PlaywrightTest("browsertype-connect.spec.ts", "should be able to reconnect to a browser")]
-        [Ignore("SKIP WIRE")]
-        public void ShouldBeAbleToConnectToBrowserAsync()
+        public async Task ShouldBeAbleToConnectToBrowserAsync()
         {
+            {
+                var browser = await BrowserType.ConnectAsync(_browserServer.WSEndpoint);
+                var context = await browser.NewContextAsync();
+                var page = await context.NewPageAsync();
+                var result = await page.EvaluateAsync("11 * 11");
+                Assert.AreEqual(result.ToString(), "121");
+                await browser.CloseAsync();
+            }
+            {
+                var browser = await BrowserType.ConnectAsync(_browserServer.WSEndpoint);
+                var context = await browser.NewContextAsync();
+                var page = await context.NewPageAsync();
+                await page.GotoAsync(Server.EmptyPage);
+                Assert.AreEqual(Server.EmptyPage, page.Url);
+                await browser.CloseAsync();
+            }
         }
 
         [PlaywrightTest("browsertype-connect.spec.ts", "should be able to connect two browsers at the same time")]
-        [Ignore("SKIP WIRE")]
-        public void ShouldBeAbleToConnectTwoBrowsersAtTheSameTime()
+        public async Task ShouldBeAbleToConnectTwoBrowsersAtTheSameTime()
         {
+            var browser1 = await BrowserType.ConnectAsync(_browserServer.WSEndpoint);
+            Assert.AreEqual(browser1.Contexts.Count, 0);
+            await browser1.NewContextAsync();
+            Assert.AreEqual(browser1.Contexts.Count, 1);
+
+            var browser2 = await BrowserType.ConnectAsync(_browserServer.WSEndpoint);
+            Assert.AreEqual(browser2.Contexts.Count, 0);
+            var context2 = await browser2.NewContextAsync();
+            Assert.AreEqual(browser1.Contexts.Count, 1);
+            Assert.AreEqual(browser2.Contexts.Count, 1);
+
+            await browser1.CloseAsync();
+            Assert.AreEqual(browser2.Contexts.Count, 1);
+
+            var page2 = await context2.NewPageAsync();
+            var result = await page2.EvaluateAsync("11 * 11");
+            Assert.AreEqual(result.ToString(), "121");
+            await browser2.CloseAsync();
         }
 
         [PlaywrightTest("browsertype-connect.spec.ts", "disconnected event should be emitted when browser is closed or server is closed")]
-        [Ignore("SKIP WIRE")]
-        public void DisconnectedEventShouldBeEmittedWhenBrowserIsClosedOrServerIsClosed()
+        public async Task DisconnectedEventShouldBeEmittedWhenBrowserIsClosedOrServerIsClosed()
+        {
+            var browserOne = await BrowserType.ConnectAsync(_browserServer.WSEndpoint);
```

Comment:
This test does not seem to be on par with the one in https://github.com/microsoft/playwright/blob/main/tests/browsertype-connect.spec.ts.

We should port all the tests as is to ensure connect is flawless.